### ISSUE TITLE
ADD a `canResize: true` example window

### ISSUE THREAD

**Issue by allanlaal** [nativeshell/examples#16]
currently all the opened windows have WindowStyle `canResize: false`, which will leave users thinking nativeshell does not support window resizing :confused:

**Comment by knopp** [comment]
Not quite sure what you mean, the main window is resizable.

**Comment by allanlaal** [comment]
yes, but the new windows feel like nativeshell magic.  there are frameworks that can only make half-assed windows (like whatever IntelliJ is using)

**Comment by allanlaal** [comment]
perhaps an explicit resizeable modal?
